packages/patternfly-4/gatsby-theme-patternfly-org/components/autoLinkHeader/autoLinkHeader.js

```diff
@@ -19,6 +19,7 @@ export const AutoLinkHeader = ({
   size,
   headingLevel,
   children,
+  metaText,
```

Comment:
This prop is essentially `children`, there's no need for it.

Reply:
Using `children` produces the following fragment when selecting the anchor

<img width="252" alt="Screen Shot 2020-02-10 at 1 13 30 PM" src="https://user-images.githubusercontent.com/5942899/74177239-5acba900-4c07-11ea-8021-ad34a4d442c2.png">

Reply:
Ah, good point! I forgot we serialize children for the anchor link.